README.md

```diff
@@ -5,6 +5,20 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/openshift/assisted-service)](https://goreportcard.com/report/github.com/openshift/assisted-service)
 [![License Apache](https://img.shields.io/github/license/openshift/assisted-service)](https://opensource.org/licenses/Apache-2.0)
 
+## About
+
+This repository provides a service that installs OpenShift.  The service exposes either a REST API, or it can be deployed as an Operator where it exposes a Kubernetes-native API via Custom Resources.  A [UI](https://github.com/openshift-assisted/assisted-ui) is available that uses the REST API.
+
+The Assisted Service can currently install clusters with highly-available control planes (3 hosts and above) and can also install Single-Node OpenShift (SNO).  Highly available clusters are configured to use OpenShift's `bare-metal` platform, while SNO uses `none`.
```

Comment:
Is there a link explaining the different platform configurations? What about VMs?

Reply:
Added links for platforms.  We don't support other platforms.